src/assets/styles/_variables.scss

```
@@ -0,0 +1,4 @@
+$bg-color: #f8f9ff;
+$primary-color: #7a8dfd;
+$color-white: #ffffff;
+$border-color: #dde2ff;
```

Common practice is to use color as prefix
```suggestion
$color-bg: #f8f9ff;
$color-primary: #7a8dfd;
$color-white: #ffffff;
$color-border: #dde2ff;
```